bool set_sleeping
---
bool success